### ISSUE TITLE
[Adaptive Cards- Import- Input Form with Validation]: 'Tell us about yourself' text above the input field is not visible after resizing the screen.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7262]
https://microsoft.visualstudio.com/OS/_workitems/edit/34110137

**Comment by vagpt** [comment]
This issue is not repro hence closing this issue.
